apps/client/src/components/compounds/SavingsTargetCard/index.tsx

```diff
@@ -0,0 +1,59 @@
+import styles from './savingsTarget.module.scss'
+import { Text } from 'components'
+
+function formatNaira(amount: number): string {
+  const formattedAmount = amount.toLocaleString('en-NG', {
+    style: 'currency',
+    currency: 'NGN',
+  })
+  return formattedAmount.replace('NGN', '₦')
+}
```

Comment:
i think this exist in the codebase already somewhere...
if yes, move this to the utility function and import it here
also do same for where it was used within the code